client/src/components/grants/inputs.tsx

```diff
@@ -250,3 +250,53 @@ export function Select({
     </div>
   );
 }
+
+export function CustomSelect({
```

Comment:
I wonder if we can make the existing `Select` input work for this use case without creating a custom one. Like iterating through `options` and creating the desired value-key pairs before passing it to the `Select` component

Reply:
I want to eventually replace using the react-select but wanted a working version done. If we use the react-select we will have to rename the Select also. Eventually they can be merged and used all over.